airflow/operators/swarm_operator.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you move this file to contrib? We do not add new operators to airflow.operators.

Reply:
@mik-laj I didn't know that, thanks for pointing out - I'll get to it.

Also, can you please tell me what [Travis Pylint](https://travis-ci.org/apache/airflow/jobs/550935644) expects when it says `Too many instance attributes (18/15) (too-many-instance-attributes)` and `Specify string format arguments as logging function parameters`?
I see other operator classes with much more attributes than **15** and logging strings the way I am doing in this PR - `self.log.info('Service started: %s' % str(self.service))`.

Reply:
In case of ``too-many-instance-attributes``:
Is it not possible to group multiple similar options and passing as namedtuple?
For example:
````
class DockerSwarmTlsOption(typings.namedTuple):
    ca_cert: str
    client_cert: str,
    ....
```

If this is not possible then you should use `pylint disable = too-many-instance-attributes`.

Can you show examples when logging is used incorrectly?
https://github.com/apache/airflow/pull/4804
https://github.com/apache/airflow/pull/5466
I work hard to eliminate these problems, but I could have missed something.

Not all files are checked by pylint, because we are in the adaptation period. 
https://github.com/apache/airflow/blob/master/scripts/ci/pylint_todo.txt
https://issues.apache.org/jira/browse/AIRFLOW-4364
All new files must comply with the pylint.